
Unhandled Exception:
System.Runtime.CompilerServices.RuntimeWrappedException: An object that does not derive from System.Exception has been wrapped in a RuntimeWrappedException.
  at Test.Main.entry () [0x00000] in <551653c51c844dd7aa0ae6d0b1c30b5e>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.Runtime.CompilerServices.RuntimeWrappedException: An object that does not derive from System.Exception has been wrapped in a RuntimeWrappedException.
  at Test.Main.entry () [0x00000] in <551653c51c844dd7aa0ae6d0b1c30b5e>:0 
